tests/test_all.py

```diff
@@ -686,6 +742,25 @@ def test_check_returns_false_with_no_args(self):
         response = self.client.get(reverse('invite:check'))
         self.assertFalse(json.loads(response.content)['taken'])
 
+    def test_check_invite_duplicate_exists(self):
+        self.client.login(username='superuser', password='superuser')
+        self.client.post(
+            reverse('invite:invite'),
+            {
+                'form-MAX_NUM_FORMS': [''],
+                'form-INITIAL_FORMS': ['0'],
+                'form-TOTAL_FORMS': ['2'],
```

Comment:
Likely of no consequence, but for appearances, this could be
```suggestion
                'form-TOTAL_FORMS': ['1'],
```